src/request/export/index.html

```diff
@@ -167,7 +178,7 @@ <h1 data-i18n="recovery-words-title" class="nq-h1">Write these 24 words on a pie
                 </div>
 
                 <div class="page-footer">
-                    <button class="to-validate-words nq-button light-blue" data-i18n="continue">Continue</button>
+                    <button class="to-validate-words nq-button light-blue" data-i18n="recovery-words-validate">validate words</button>
```

Comment:
Capitalization? I know it's uppercased in the button, but grammatically?
```suggestion
                    <button class="to-validate-words nq-button light-blue" data-i18n="recovery-words-validate">Validate words</button>
```